src/HotChocolate/Spatial/demo/Spatial.Demo/Startup.cs

```diff
@@ -0,0 +1,46 @@
+using HotChocolate;
+using HotChocolate.Data.Filters;
+using HotChocolate.Data.Filters.Expressions;
+using HotChocolate.Data.Spatial.Filters;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Spatial.Demo
+{
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContextPool<ApplicationDbContext>(
+                    options => options.UseNpgsql(
+                        "Database=opensgid;Host=opensgid.agrc.utah.gov;Username=agrc;Password=agrc",
+                        o => o.UseNetTopologySuite()))
+                .AddGraphQLServer()
+                .AddSpatialTypes()
+                .AddFiltering(
+                    x => x
+                        .AddDefaults()
```

Comment:
We should add another method like AddDefaults.... AddSpatialSupport or something like this that does this all for us ....

`.AddFiltering(x => x.AddSpatialDefaults());` something like that